File: php/testcode/benchmark_test_00764.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00764(BenchmarkRequest $req): BenchmarkResponse {
    putenv('GOOGLE_APPLICATION_CREDENTIALS=' . getenv('GCP_CREDS_PATH'));
    $project = $req->param('project');
    $scope = 'https://www.googleapis.com/auth/cloud-platform';
    return BenchmarkResponse::ok('gcp auth configured for project: ' . $project);
}

File: rust/testcode/benchmark_test_00764.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let content = req.param("content");

    let html = format!("<iframe src=\"data:text/html,{}\"></iframe>", content);

    super::shared::BenchmarkResponse::ok(&html)
}
